PROPOSAL.md

```diff
@@ -0,0 +1,181 @@
+
+service : {
+    icrc1_metadata : () -> (vec record { text; Value; }) query;
+    icrc1_name : () -> (text) query;
+    icrc1_symbol : () -> (text) query;
+    icrc1_decimals : () -> (nat8) query;
+    icrc1_fee : () -> (nat) query;
+    icrc1_total_supply : () -> (nat) query;
+    icrc1_minting_account : () -> (opt Account) query;
+    icrc1_balance_of : (Account) -> (nat) query;
+    icrc1_transfer : (TransferArgs) -> (variant { Ok : nat; Err : TransferError });
+    icrc1_supported_standards : () -> (vec record { name : text; url : text }) query;
+}
+```
+
+## Core traits of the standard
+
+### Two-component account structure
+
+ICRC-1 identifies [accounts](https://github.com/dfinity/ICRC-1#account) with a [principal](https://internetcomputer.org/docs/current/references/ic-interface-spec/#principal) and an optional subaccount.
+A subaccount is an arbitrary 32-byte blob.
+Below are the arguments against clumping the owner and the subaccount irreversibly on the interface level:
+
+* Having address components unobscured is crucial for interoperability because it allows for more compliant implementations.
+  For example, the ICP ledger can turn the address components into an [account identifier](https://internetcomputer.org/docs/current/references/ledger#_accounts), and ledgers using principals to identify accounts can use [derived principals](https://internetcomputer.org/docs/current/references/ic-interface-spec/#id-classes) instead.
+  Enforcing a specific encoding would harm interoperability.
+* The ability to pass account components unaltered removes the need to link complex account identifier computation libraries to each ledger client.
+* Having owners and subaccounts in the transaction log enables new kinds of applications.
+  For example, it becomes possible to build auxiliary smart contracts acting as replicators indexing the transaction log or sending out notifications (note, however, that the standard [does not specify how to access the transaction log](#unspecified-transaction-structure)).
+* Implementations storing unaltered principals and subaccounts allow clients to recover their state fully by replaying records from the transaction log.
+* Two-component structure maps well to the [Rosetta API specification of account identifiers](https://www.rosetta-api.org/docs/api_identifiers.html#account-identifier).
+
+### Namespaces
+
+All ICRC-1 [methods](https://github.com/dfinity/ICRC-1#methods) have the `icrc1_` prefix.
+This feature allows existing smart contracts to implement ICRC-1 methods without introducing name clashes.
+
+### Unspecified transaction structure
+
+ICRC-1 _does not_ specify the structure of transactions or provide methods for fetching them.
+That deliberate design choice enables backward compatibility with existing token ledgers, such as the ICP ledger.
+The Ledger & Tokenization working group plans to work on an _optional_ core extension specifying the interface for fetching transactions.
```

Comment:
```suggestion
The Ledger & Tokenization working group plans to work on an _optional_ extension specifying the interface for fetching transactions.
```
The word 'core' and the word 'extension' are opposites and putting them in the same-phrase is contradictory.